scripts/superdesk-authoring/authoring.js

```diff
@@ -893,6 +906,12 @@
                 $scope.reject();
             }
         };
+
+        $scope.onChange = function(renditionName, cropData) {
+            $scope.data.cropData[renditionName] = angular.extend({}, cropData);
+            $scope.data.isDirty = true;
+            $scope.$apply();
```

Comment:
better to put scope changes here inside a function passed to $apply for error catching purposes.